AnselemOdims/Loaner
# #165541365 Set up the express framework

### What does this PR do?
Sets up the Express framework

#### Description of Task to be completed?
- Run npm init
- Edit the package.json file
- Install the express framework using "npm i --save express"
- Include a .gitignore file

#### How can it be manually tested
- Clone this repository
- cd into the directory
- Do git checkout ch-set-up-express-165541365
- Then cd into package.json 
- check dependencies

#### What are the relevant pivotal tracker stories?
[#165541365](https://www.pivotaltracker.com/n/projects/2328795, "set-up express framework")
